README.md

```diff
@@ -36,6 +36,27 @@ yarn build
 
 Strapi gives you many possible deployment options for your project including [Strapi Cloud](https://cloud.strapi.io). Browse the [deployment section of the documentation](https://docs.strapi.io/dev-docs/deployment) to find the best solution for your use case.
 
+# 🚀 Getting started with Gitpod
+Gitpod provides a fully automated development environment for your Strapi project, and the development environment is set up with just a single click. Follow these steps to get started:
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/distributeaid/aggregated-public-information)
+
+1. Click the ``Open in Gitpod`` button above. Note: you'll need to have an account on [Gitpod](https://gitpod.io/login/) before proceeding with the next steps (this requires a GitHub account)..
+2. Click the `Continue` button.
+3. Relax, a development environment is being set up for you in the first terminal. There's currently a bug that will stop the setup from being completed, but we'll fix that in the next steps.
+4. Create a new file called `.env` to store the environment variables.
+5. Copy and paste the content from the `.env.example` file into the `.env` file you just created.
+6. Open a new terminal and paste this code inside `openssl rand -base64 16`. This generates a secret key that will be used inside the env file.
+
+**Note:** Do not copy or use the generated keys that have `plus` or `forward slash` in them. Only copy the ones with `numbers`, `alphabets` and `equals`.
```

Comment:
**Nitpick:** Can we make this a bullet under step 6, instead of breaking the numbering flow?

Reply:
Sure thing